docs/update.md

```diff
@@ -82,6 +84,44 @@ You can upgrade the Operator and CRD as follows, considering the Operator uses
         deployment "percona-postgresql-operator" successfully rolled out
         ```
 
+### Upgrade via Helm
+
+If you have [installed the Operator using Helm](helm.md), you can upgrade the
+Operator with the `helm upgrade` command.
+
+1. In case if you installed the Operator with no [customized parameters :octicons-link-external-16:](https://github.com/percona/percona-helm-charts/tree/main/charts/pg-operator#installing-the-chart), the upgrade can be done as follows: 
+
+    ``` {.bash data-prompt="$" }
+    $ helm upgrade my-operator percona/pg-operator --version {{ release }}
+    ```
+
+    The `my-operator` parameter in the above example is the name of a [release object :octicons-link-external-16:](https://helm.sh/docs/intro/using_helm/#three-big-concepts)
+    which which you have chosen for the Operator when installing its Helm chart.
+
+    If the Operator was installed with some [customized parameters :octicons-link-external-16:](https://github.com/percona/percona-helm-charts/tree/main/charts/pg-operator#installing-the-chart), you should list these options in the upgrade command.
+    
+    
+    !!! note
+    
+        You can get list of used options in YAML format with the `helm get values my-operator -a > my-values.yaml` command, and this file can be directly passed to the upgrade command as follows:
+
+        ``` {.bash data-prompt="$" }
+        $ helm upgrade my-operator percona/pxc-operator --version {{ release }} -f my-values.yaml
+        ```
+
+2. Update the [Custom Resource Definition :octicons-link-external-16:](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+    for the Operator, taking it from the official repository on Github, and do
+    the same for the Role-based access control:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply --server-side -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/crd.yaml
+    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/rbac.yaml -n postgres-operator
```

Comment:
we are not specifying the namespace in other commands, so we have to specify in others that need or delete this from here